tlslite/utils/rsakey.py

```diff
@@ -87,6 +91,167 @@ def hashAndVerify(self, sigBytes, bytes):
         result2 = self.verify(sigBytes, prefixedHashBytes2)
         return (result1 or result2)
 
+    def MGF1(self, mgfSeed, maskLen, hAlg):
+        """Generate mask from passed-in seed.
+
+        This generates mask based on passed-in seed and output maskLen.
+
+        @type mgfSeed: L{bytearray}
+        @param mgfSeed: Seed from which mask will be generated.
+
+        @type maskLen: int
+        @param maskLen: Wished length of the mask, in octets
+
+        @rtype: L{bytearray}
+        @return: Mask
+        """
+        hashLen = getattr(hashlib, hAlg)().digest_size
+        if maskLen > (2 ** 32) * hashLen:
+            raise MaskTooLongError("Mask generating function failed")
+        T = bytearray()
+        end = (Poly1305.divceil(maskLen, hashLen))
```

Comment:
I see just one test case and it doesn't use `maskLen` that is equal multiple of `hashLen`

is it covered by one of the KATs?